apps/redi-talent-pool/src/components/organisms/company-profile-editables/EditableJobPostings.tsx

```diff
@@ -73,6 +73,11 @@ export function EditableJobPostings({
     startEditing(id)
   }
 
+  const renderTimestamp = (expiresAt) =>
+    `Expires in ${formatDistance(new Date(expiresAt), new Date(), {
+      addSuffix: true,
```

Comment:
@helloanil, the value for the timestamp here is a bit off. It displays the word `in` two times. 
![image](https://github.com/talent-connect/connect/assets/51786805/ebcb4c0a-1213-444d-b2b7-cee64f252ed4)